tasks/clean_engine.yml

```diff
@@ -23,48 +23,50 @@
     - name: Set force remove flag to false for non master domains
       set_fact: dr_force=False
 
-    - name: Set query for remove valid non master domains
-      # TODO: Filter only data and iso types
-      set_fact: dr_query_domain_search='status = active and type != cinder or status = maintenance and type != cinder or status = unattached and type != glance and type != cinder'
+    # Set all the queries suffix to fetch a storage domain in a specific status.
+    # Note: Export storage domain is not supported and should not be part of storage mapping
+    - name: Set query suffix for active storage domains
```

Comment:
This can be:

```
-name Setup queries for storagedomains
 set_fact:
   dr_active_domain_search='status = active and type != cinder'
   dr_unattached_domain_search='status = unattached and type != cinder and type != glance'
   ....

Reply:
I only saw that comment now, added a commit which fixes that in PR 43

Thanks